cli.js

```diff
@@ -3,18 +3,31 @@
 var meow = require('meow');
 var gitSemverTags = require('./');
 
-meow({
+var args = meow({
   help: [
     'Usage',
-    '  git-semver-tags'
+    '  git-semver-tags',
+    'Options',
+    ' --lerna, -l parse lerna style git tags',
+    ' --package, -p when listing lerna style tags, filter by a package'
   ]
 });
 
+// the package argument only makes sense
+// when used in the context of lerna-style git-tags.
+if (args.flags.package && !args.flags.lerna) {
```

Comment:
Would it make more sense to have this inside the actual tool, since now it will miss faulty programmatic usage?